grantnav/frontend/urls.py

```diff
@@ -36,7 +36,7 @@
     url(r'^take_down_policy', TemplateView.as_view(template_name='take_down_policy.html'), name='take_down_policy'),
     url(r'^terms', TemplateView.as_view(template_name='terms.html'), name='terms'),
     url(r'^help', RedirectView.as_view(url="https://help.grantnav.threesixtygiving.org/"), name="help"),
-    url(r'^developers', TemplateView.as_view(template_name='developers.html'), name='developers'),
+    url(r'^developers', RedirectView.as_view(url="https://help.grantnav.threesixtygiving.org/en/latest/developers.html"), name='developers'),
```

Comment:
@odscjames I understand this, but tbh I think we'll completely forget about the rationale here. Can we either leave a comment or (my preferred option) just remove it?

Reply:
We can only remove if we are sure nothing links to /developers, seems easy and cheap to leave it in. I'll add a comment